src/applyLockMass.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const analyseMF = require('chemcalc').analyseMF;
+
+/**
+ * Recalculates series for GC/MS with lock mass
+ * @param {Chromatogram} chromatogram - GC/MS chromatogram where make the peak picking
+ * @param {string|Array<string>} molecularForm - Reference molecular formula(s)
+ * @return {{tic: Array<number>, ms: Array<Array<number>>, time: Array<number>}}
+ */
+function applyLockMass(chromatogram, molecularForm) {
+    // allows molecularForm as string or array
+    if (typeof molecularForm === 'string') {
+        molecularForm = [molecularForm];
+    }
+
+    // calculate the mass reference values
+    const referenceMass = molecularForm.map((mf) => analyseMF(mf).em);
+
+    const ms = chromatogram.findSerieByName('ms').data;
+    if (!ms) {
+        throw new Error('The mass serie must be defined');
+    }
+    const time = chromatogram.getTimes();
+
+    // check where is the reference values
+    var oddReference;
```

Comment:
You should not try to determine it automatically I think. Just use an option for this and the default value is the one we would like for our problem